src/routes/LicensesRoute.js

```diff
@@ -124,6 +134,41 @@ class LicensesRoute extends React.Component {
     }
   }
 
+  downloadBlob = () => (
+    blob => {
+      const url = window.URL.createObjectURL(blob);
+      const a = document.createElement('a');
+      a.href = url;
+      a.download = 'compare_terms.csv';
+      document.body.appendChild(a);
+      a.click();
+      a.remove();
+    }
+  )
+
+  handleCompareLicenseTerms = (payload) => {
+    const { stripes: { okapi } } = this.props;
+
+    return fetch(`${okapi.url}/licenses/licenses/compareTerms`, {
+      method: 'POST',
+      headers: {
+        'X-Okapi-Tenant': okapi.tenant,
+        'X-Okapi-Token': okapi.token,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(payload),
+    }).then(response => {
+      if (response.ok) {
+        return response.blob();
+      } else {
+        throw new Error('failed');
+      }
+    })
+      .then(this.downloadBlob())
+      .catch(error => {
```

Comment:
Will take this off if we do not require any error handling... added this as a placeholder